### DISCUSSION TITLE
Need help `command 'latex-workshop.build' not found`

### DISCUSSION THREAD

**Discussion by amirtadayon** [James-Yu/LaTeX-Workshop#3120] [Q&A]
Hi

I just updated my VS Code to the latest release October 2020 (version 1.51). Now, I have an issue running latex using LaTeX-Workshop. When I want to compile my text file, I receive the message `command 'latex-workshop.build' not found`. Similar message `command 'latex-workshop.view' not found` appears when I want to view the corresponding PDF (which is already compiled using, e.g.,  TeXShop).
Please advise how to get the extension to work.

Thanks

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.

**Comment by amirtadayon** [comment]
> 
> 
>     * Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md).
> 
>     * Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.
> 
> 
> ![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)
> 
> When all of the above is done, we will reopen this issue if needed.

As shown in the attached screen shot, I do not have the TEX tab in the Activity Bar any more. 
Please let me know how I can get the log required for completing the issue template.
![VSCode_LaTeX](https://user-images.githubusercontent.com/5162994/98843537-1af55480-2419-11eb-986a-39f916817b3b.PNG)

**Comment by jlelong** [comment]
Completely uninstall the extension and reinstall it. If using the `uninstall` menu may not enough, see https://stackoverflow.com/questions/36746857/completely-uninstall-vs-code-extensions.
